phoenix-pherf/src/test/java/org/apache/phoenix/pherf/workload/MultiThreadedRunnerTest.java

```diff
@@ -0,0 +1,102 @@
+package org.apache.phoenix.pherf.workload;
+
+import org.apache.hadoop.hbase.util.Pair;
+import org.apache.phoenix.pherf.configuration.Query;
+import org.apache.phoenix.pherf.configuration.Scenario;
+import org.apache.phoenix.pherf.configuration.XMLConfigParser;
+import org.apache.phoenix.pherf.result.DataModelResult;
+import org.apache.phoenix.pherf.result.ThreadTime;
+import org.apache.phoenix.pherf.rules.RulesApplier;
+import org.apache.phoenix.util.DefaultEnvironmentEdge;
+import org.apache.phoenix.util.EnvironmentEdgeManager;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.sql.ResultSet;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+public class MultiThreadedRunnerTest {
+
+    @Test
+    public void testExpectedRowsMismatch() throws Exception {
+        XMLConfigParser mockParser = Mockito.mock(XMLConfigParser.class);
+        DataModelResult mockDMR = Mockito.mock(DataModelResult.class);
+        RulesApplier mockRA = Mockito.mock(RulesApplier.class);
```

Comment:
All these common mocked objects can be initialized just once in a `BeforeClass` method and then re-used in each test, to avoid code repetition.